importers/djornl/parser.py

```diff
@@ -167,22 +286,29 @@ def load_cluster_data(self):
 
         # index of nodes
         node_ix = {}
-
-        cluster_paths = self.config('_CLUSTER_PATHS')
-        for (cluster_label, path) in cluster_paths.items():
-            with open(path) as fd:
-                csv_reader = csv.reader(fd, delimiter='\t')
-                for row in csv_reader:
-                    if len(row) > 1:
-                        # remove the 'Cluster' text and replace it with cluster_label
-                        cluster_id = cluster_label + ':' + row[0].replace('Cluster', '')
-
-                        node_keys = row[1:]
-                        for key in node_keys:
-                            if key not in node_ix:
-                                node_ix[key] = [cluster_id]
-                            elif cluster_id not in node_ix[key]:
-                                node_ix[key].append(cluster_id)
+        for file in self.config('cluster_files'):
+            cluster_label = file['cluster_prefix']
+            headers = []
+            file_parser = self.parser_gen(file)
+
+            while True:
+                try:
+                    (line_no, cols) = next(file_parser)
+                except StopIteration:
+                    break
+
+                if len(headers) == 0:
+                    headers = cols
+                    continue
+
+                # remove the 'Cluster' text and replace it with cluster_label
+                cluster_id = cluster_label + ':' + cols[0].replace('Cluster', '')
+                node_keys = [n.strip() for n in cols[1].split(',')]
```

Comment:
asked the DJORNL people to change the format of the cluster files so they're valid TSV/CSV.